src/splitInteger.test.js

```diff
@@ -1,21 +1,29 @@
-'use strict';
+"use strict";
 
-const splitInteger = require('./splitInteger');
+const splitInteger = require("./splitInteger");
 
-test(`should split a number into equal parts
+it(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
+  const result = splitInteger(8, 1);
 
+  expect(result).toEqual([8]);
 });
```

Comment:
If you check the part is divisible by two equal parts, then you need to show it in your example. I suggest:
```
   const result = splitInteger(8, 2);

   expect(result).toEqual([4, 4]);
  ```